src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -50,8 +51,8 @@
         items            (filter-and-sort-items-by-tab selected-tab unfiltered-items)]
     (if (empty? items)
       [common.home/empty-state-image
-       {:selected-tab selected-tab
-        :tab->content empty-state-content}]
+       {:selected-tab :tab/contacts
+        :tab->content (empty-state-content)}]
```

Comment:
Is this change ok?
we are ignoring `selected-tab` and setting `:tab/contacts` as a fixed value :thinking: Maybe all the tabs will show the same empty-state image.

Reply:
@jo-mut, here you should be passing the theme along too as ideally we are not getting theme value from these global ratoms anymore. you will need to wrap this component in the quo.theme/with-theme HOC.